django/text_extractor/views.py

```diff
@@ -88,26 +87,26 @@ def submit_document(request):
                     if idx > 0:  # Exclude TOC from file names to merge
                         file_names_to_merge.append(file_name)
                 else:
-                    file_id = f"{user_name}_{current_time}_OCR_{input_name}.pdf"
-                    text_id = f"{user_name}_{current_time}_{input_name}.txt"
-                    file_name = f"OCR_{input_name}.pdf"
-                    text_name = f"OCR_{input_name}.txt"
+                    file_name = f"{input_name}_OCR.pdf"
+                    text_name = f"{input_name}_OCR.txt"
 
-                    content_file = ContentFile(pdf_bytes.getvalue(), name=file_id)
-                    content_text = ContentFile(txt_file, name=text_id)
+                    content_file = ContentFile(
+                        pdf_bytes.getvalue(), name=shorten_input_name(file_name)
```

Comment:
We use the new generated (short, unique) filename only here, for the file that is actually written to /media folder.